SupportBank/Program.cs

```diff
@@ -1,12 +1,46 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using CsvHelper;
 
 namespace SupportBank
 {
     class Program
     {
+        private static Dictionary<string,Account> _accounts;
+        
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            _accounts = new Dictionary<string, Account>();
+            CSVReader();
+            foreach (var account in _accounts.Values)
+            {
+                Console.WriteLine($"Name: {account.Name} -- Pending Account Balance: {account.BalanceDue}");
+            }
+        }
+
+        static void CSVReader()
```

Comment:
This method name doesn't really explain what the method does, could you give it a more descriptive name? 🙂